src/org/sosy_lab/java_smt/SolverContextFactory.java

```diff
@@ -58,6 +60,28 @@ public enum Solvers {
     YICES2
   }
 
+  public enum Logics {
```

Comment:
The logics are only required for OpenSMT. Do we really need them to be in the context-factory? Would a ConfigurationOption in the OpenSMT-context be sufficient?

Reply:
We thought that other solvers may also benefit from a way to manually select a specific logic. However, I can look into moving it to the OpenSMT context later today.